.github/workflows/pull-request-checks.yaml

```diff
@@ -426,8 +426,8 @@ jobs:
           Set-Location build
           ctest -V -L CORE -C Release . -j2
 
-  check-vs-2019-make-build-and-test:
-    runs-on: windows-2019
+  check-vs-2022-make-build-and-test:
```

Comment:
@peterschrammel Are there any downstream users of CBMC that we still need to support on VS 2019?

Reply:
Checking...

Reply:
Yes, there are. I think it's fine to upgrade the make job to VS2022 as long as we keep the VS2019 cmake job.